gamestonk_terminal/stocks/behavioural_analysis/ba_controller.py

```diff
@@ -890,6 +896,56 @@ def call_historical(self, other_args: List[str]):
             else:
                 print("No ticker loaded. Please load using 'load <ticker>'\n")
 
+ 
+    @try_except
+    def call_his(self, other_args: List[str]):
+        """Process his command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="his",
+            description="Plot historical sentiment data of RHI and AHI by hour",
+        )
+
+        parser.add_argument(
+            "-s",
+            "--start",
+            type=valid_date,
+            default=(datetime.now() - timedelta(days=7)).strftime("%Y-%m-%d"),
+            dest="start",
+            help="The starting date (format YYYY-MM-DD) of the stock. Default: 7 days ago",
+        )
+
+        parser.add_argument(
+            "-e",
+            "--end",
+            type=valid_date,
+            default=(datetime.now().strftime("%Y-%m-%d")),
+            dest="end",
+            help="The ending date (format YYYY-MM-DD) of the stock. Default: today",
+        )
+
+        parser.add_argument(
+            "-n",
+            "--number",
+            default=100,
+            type=check_positive,
+            dest="number",
+            help="Limit number of returned results. Defaul: 100",
+        )
+
+        ns_parser = parse_known_args_and_warn(parser,
+        other_args, EXPORT_BOTH_RAW_DATA_AND_FIGURES)
+
+        if ns_parser:
+            sentimentinvestor_view.display_historical(
+                    ticker=self.ticker,
+                    start=ns_parser.start,
+                    end=ns_parser.end,
+                    limit=ns_parser.number,
+                    export=ns_parser.export
+            )
```

Comment:
So we should also allow for the user to print out some data to the terminal.  The parse_known_args_and_warn can add the following: (raw=True, limit=10).  This will add
`ns_parser.raw` and` ns_parser.limit` by default.